var/spack/repos/builtin/packages/gdal/package.py

```diff
@@ -155,6 +157,43 @@ def configure_args(self):
             '--with-libjson-c={0}'.format(spec['json-c'].prefix),
         ]
 
+        if spec.satisfies('@2:'):
```

Comment:
Just to confirm, none of these flags are available in 1.11.5 but all of them are available in 2.0.0? Just making sure they weren't added sporadically.

Reply:
To be honest, I did not check any 2.x versions when I introduced these checks as it kept the 2.x dependencies the same. I have now gone through all available versions and adapted the checks. Looks like some flags were introduced in 2.1 and 2.3. Moreover, only 2.3 actually compiles with recent versions of json-c.